.travis.yml

```diff
@@ -21,10 +21,11 @@ cache:
 before_script:
   # default $SBT_OPTS is irrelevant to sbt lancher
   - unset SBT_OPTS
+  # ensure the build errors if dependencies aren't available
+  - travis_retry ./bin/sbt ++$TRAVIS_SCALA_VERSION update
```

Comment:
not easy to find, but yeah 3 tries

http://docs.travis-ci.com/user/build-timeouts/